core/domain/story_domain.py

```diff
@@ -242,6 +242,10 @@ def validate(self):
                 'Expected node ID to be a string, received %s' %
                 self.id)
 
+        if 'node_' not in self.id:
```

Comment:
Define a constant for this and refer to it where it's used (NODE_ID_PREFIX perhaps).

Also we should be validating more carefully. I.e. check that this string occurs at the _start_ and that it is followed by a number?

Reply:
Done, the check for number is done in the validation of whether number is greater than next_node_id.